src/swing-card-component.ts

```diff
@@ -31,4 +31,10 @@ export class SwingCardComponent {
   getCard(): Card {
     return this.swingStack.stack.getCard(this.getNativeElement());
   }
+
+  destroyCard() {
+		this.swingStack.cards.pop();
```

Comment:
@Sparkomatic the implementation is ambiguous - when I call destroyCard on any card, this will remove the last card. So either we should move this onto the stack, or change the implementation to actually destroy the card which its call on. Let me know what you think.